@xen-orchestra/web-core/lib/components/LegendTitle.vue

```diff
@@ -0,0 +1,39 @@
+<!-- v1.0 -->
+<template>
+  <div class="legend-title typo c3-semi-bold">
+    {{ title }}
+    <UiIcon
+      v-tooltip="{
+        placement: 'top',
+        content: tooltip ? $t(tooltip) : '',
+      }"
+      :icon="tooltip ? icon : undefined"
+      color="info"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+
+defineProps<{
+  title: string
+  tooltip?: string
+  icon?: IconDefinition
+}>()
```

Comment:
- Remove `title` prop in favor of a slot
- Rename `tooltip` to `iconTooltip` to be explicit that the tooltip will be applied on the icon only, not on the whole label. It also implicitly gives advice to the developer that "no icon = no tooltip".

```suggestion
defineProps<{
  icon?: IconDefinition
  iconTooltip?: string
}>()
```